packages/simulation/tests/convertNFAtoDFA.test.ts

```diff
@@ -11,6 +11,8 @@ import convertSingleTrapState from './graphs/convertSingleTrapState.json'
 import convertSingleTrapStateDFA from './graphs/convertSingleTrapStateDFA.json'
 import infiniteLoop from './graphs/infiniteLoop.json'
 import infiniteLoopDFA from './graphs/infiniteLoopDFA.json'
+import issue445Expected from './graphs/issue445DFA.json'
+import issue445Input from './graphs/issue445NFA.json'
```

Comment:
Weird naming but go off king 🤪 (I'd name it something more relevant to the contents of the machine, someone in the future isn't really gonna know what issue 445 is)

Reply:
Smh people should know every issue number in their head before even knowing what automatarium is 😤

But ye will update that name, and chuck in a test case for the [a-z] thing